.github/ISSUE_TEMPLATE/copyright-edit.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Copyright Edit
```

Comment:
This also shouldn't say "copyright". And what is the difference between this template and the "Content Edit Request"? These seem like duplicates